rpcd/playbooks/roles/rpc_maas/templates/nova_cloud_stats_check.yaml.j2

```diff
@@ -0,0 +1,31 @@
+type: agent.plugin
+label: "nova_cloud_stats_check--{{ ansible_hostname }}"
+disabled    : false
+period      : "{{ maas_check_period }}"
+timeout     : "{{ maas_check_timeout }}"
+details     :
+    file    : nova_cloud_stats.py
+    args    : ["{{ ansible_ssh_host }}"]
+alarms      :
```

Comment:
You should have three alarms here, one for memory, one for disk and one for CPU.